LogMoji/Classes/State.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  State.swift
+//  LogMoji
+//
+//  Created by Mustafa Khalil on 3/26/20.
+//
+
+import Foundation
+
+public protocol State {
+    /// Emoji that would be used by the library to print the color of the state
```

Comment:
I don't understand this comment really. What does the emoji have to do with the color of the state?